src/plugins/placeholder.ts

```diff
@@ -60,15 +61,25 @@ export default () =>
         const decorations: Decoration[] = []
 
         state.doc.descendants((node, pos, parent) => {
-          if (!node.isAtom && node.type.isBlock && node.childCount === 0) {
-            if (node.type === node.type.schema.nodes.paragraph) {
-              const text = getParagraphPlaceholderText(parent)
-              if (text) {
+          if (node.childCount === 0) {
+            if (!node.isAtom && node.type.isBlock) {
+              if (node.type === node.type.schema.nodes.paragraph) {
+                const text = getParagraphPlaceholderText(parent)
+                if (text) {
+                  decorations.push(
+                    Decoration.widget(pos + 1, placeholderWidget(text))
+                  )
+                }
+              } else {
                 decorations.push(
-                  Decoration.widget(pos + 1, placeholderWidget(text))
+                  Decoration.node(pos, pos + node.nodeSize, {
+                    class: 'empty-node',
+                  })
                 )
               }
-            } else {
+            }
+            if (node.type === node.type.schema.nodes.body) {
+              // If the body is empty, add the empty-node class
```

Comment:
no need for the changes in this file, will need to remove just atom from [body](https://github.com/Atypon-OpenSource/manuscripts-transform/blob/master/src/schema/nodes/body.ts#L24)

Reply:
@asouqi  done in https://github.com/Atypon-OpenSource/manuscripts-transform/pull/170